<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <node pkg="frame_tracking" type="frame_tracking_node" name="example" />

    <!-- Setup tracking Algorithm parameter -->
    <param name="cluster_value_" type="double" value="0.6" />
    <param name="centroid_distance_" type="double" value="1.2" />
    <param name="voxel_leaf_size_" type="double" value="0.1" />
    

</launch>
